### PR TITLE
Implement logout for keycloak

### PR DESCRIPTION
**Pull request by Thunderguts13**

### CODE REVIEW

File: app/assets/javascripts/modules/auto_logoff.js.coffee [hunk 1]
```diff
@@ -15,7 +15,7 @@ class app.AutoLogoff
 
   logoff_timer = () ->
 
-    if document.URL.indexOf('session/new') > -1
+    if document.URL.indexOf('session/new') > -1 || document.URL.indexOf('session/sso/inactive') > -1
```
Comment (psunix) [inline app/assets/javascripts/modules/auto_logoff.js.coffee:18]: create own method for this ?

isLoginPage ?

File: app/controllers/authentication/user_authenticator.rb [hunk 2]
```diff
@@ -52,6 +52,10 @@ def user_logged_in?
     raise NotImplementedError, 'implement in subclass'
   end
 
+  def logged_out_path
```
Comment (psunix) [inline app/controllers/authentication/user_authenticator.rb:55]: :+1:

Comment (psunix) [reply app/controllers/authentication/user_authenticator.rb:55]: make session_new_path default and override for keyloak ?

**Comment by Thunderguts13** [comment]
Logged in Users who visit /sso/inactive get access denied and redirected. It's "locked" with a pundit policy

File: app/assets/javascripts/modules/auto_logoff.js.coffee (hunk 1 above)
Comment (Thunderguts13) [reply app/assets/javascripts/modules/auto_logoff.js.coffee]: :+1:

File: app/controllers/authentication/user_authenticator.rb (hunk 2 above)
Comment (Thunderguts13) [reply app/controllers/authentication/user_authenticator.rb:55]: :+1:

**Pull request merged**